### ISSUE TITLE
Broadcasting to multiple clients

### ISSUE THREAD

**Issue by cb372** [hseeberger/akka-sse#139]
Hi, thanks for a very handy library!

I'm trying to take events from a Kafka queue (using akka-stream-kafka) and broadcast them to clients using SSE, so that every client receives all events. But if I connect to my server with multiple browsers, each event gets sent to only one client.

My code looks like this:

```scala
  val source = {
    val consumerSettings = ...
    val incomingTopic = ...
    Consumer
      .atMostOnceSource(consumerSettings, Subscriptions.topics(incomingTopic))
      .map { record => record.value }
  }

  val route = {
    def assets = ...
    def events = path("events") {
      get {
        complete {
          source
            .map(deliveryEventToServerSentEvent)
            .keepAlive(1.second, () => ServerSentEvent.Heartbeat)
        }
      }
    }
    assets ~ events
  }
```

I'm sure there's an easy way to set up with broadcasting/fanout using akka-streams, but I can't find it in the akka-streams docs.

I think this is quite a common use case, so maybe you could add a note to the README?

**Comment by hseeberger** [comment]
Could you please try to create a source for each client / request? 

BTW, it might be interesting to use the Last-Event-ID header, in particular for reconnects (i.e. when the client already has received some events).

**Comment by cb372** [comment]
Creating a separate Kafka consumer for each client is one option, although that's not quite how I intended things to work. Thinking about this a bit more, I guess I want it to be a publish-subscribe model, with the Kafka source acting as a publisher and each client subscribing to it. For my use case, clients don't care about events that happened before they connected; they only want to see real-time events.

I'll try turning the Kafka source into a Reactive Streams publisher (using `runWith(Sink.asPublisher)`) , and each SSE source into a subscriber (`Source.fromPublisher`) and see if that works.

Sorry for opening an issue just to think out loud 😄 . Closing.
